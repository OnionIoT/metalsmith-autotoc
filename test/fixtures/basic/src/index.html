---
title: 'hoge'
autotoc: true
template: 'layout.eco'
---
<h2>aaa</h2>

<p>paragraph</p>

<h2>bbb</h2>

<p>paragraph</p>

<h2>ccc</h2>

<p>paragraph</p>
