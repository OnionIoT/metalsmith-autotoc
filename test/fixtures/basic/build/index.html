



<ol class="toc">
  
  <li><a href="#foo-aaa">aaa</a>
    
<ol class="toc">
  
  <li><a href="#foo-bbb">bbb</a>
    
  </li>
  
  <li><a href="#foo-ccc">ccc</a>
    
  </li>
  
</ol>

  </li>
  
  <li><a href="#foo-ddd">ddd</a>
    
  </li>
  
</ol>



<h2 id="foo-aaa">aaa</h2>
<p>paragraph</p>

<h3 id="foo-bbb">bbb</h3>
<p>paragraph</p>

<h3 id="foo-ccc">ccc</h3>
<p>paragraph</p>

<h2 id="foo-ddd">ddd</h2>
<p>paragraph</p>

