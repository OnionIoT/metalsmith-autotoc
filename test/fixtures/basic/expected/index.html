<ul>

  <li><a href="#aaa">aaa</a></li>

  <li><a href="#bbb">bbb</a></li>

  <li><a href="#ccc">ccc</a></li>

</ul>

<h2 id="aaa">aaa</h2>

<p>paragraph</p>

<h2 id="bbb">bbb</h2>

<p>paragraph</p>

<h2 id="ccc">ccc</h2>

<p>paragraph</p>

